lib/story_branch.rb

```diff
@@ -200,15 +201,15 @@ def story_finish
       end
     end
 
-    def config_value key, env
+    def config_value key, env, default_value = nil
       PIVOTAL_CONFIG_FILES.each do |config_file|
         if File.exists? config_file
           pivotal_info = YAML.load_file config_file
           return pivotal_info[key] if pivotal_info && pivotal_info[key]
         end
       end
       value ||= env_required env
-      value
+      value || default_value
```

Comment:
As this is, if the value is not defined and we fallback to the default value we'll still get the error message.
It'll be more of an annoyance than anything else, because it would still work. Should the `env_required` receive the `default_value` as well so we can avoid the error message?